### PR TITLE
feat: update pageheader

### PR DESCRIPTION
**Pull request by xrkffgg**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/page-header/index.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
+import { inject } from 'vue';
 import PropTypes from '../_util/vue-types';
-import { getComponentFromProp, getOptionProps } from '../_util/props-util';
+import { getComponent, getOptionProps } from '../_util/props-util';
 import { ConfigConsumerProps } from '../config-provider';
 import ArrowLeftOutlined from '@ant-design/icons-vue/ArrowLeftOutlined';
 import Breadcrumb from '../breadcrumb';
```
Comment (xrkffgg) [inline components/page-header/index.jsx:6]: 不知道为啥报一个 breadcrumb 相关的错

File: components/page-header/index.jsx [hunk 2]
```diff
@@ -94,14 +94,16 @@ const renderChildren = (h, prefixCls, children) => {
...
+      configProvider: inject('configProvider', ConfigConsumerProps),
+    };
   },
   render(h) {
     const { getPrefixCls, pageHeader } = this.configProvider;
     const props = getOptionProps(this);
     const { prefixCls: customizePrefixCls, breadcrumb } = props;
-    const footer = getComponentFromProp(this, 'footer');
+    const footer = getComponent(this, 'footer');
     const children = this.$slots.default;
```
Comment (tangjinzhou) [inline components/page-header/index.jsx:107]: this.$slots.default && this.$slots.default()

File: components/page-header/index.jsx [hunk 3]
```diff
@@ -52,13 +52,13 @@ const renderBreadcrumb = (h, breadcrumb) => {
...
+  const tags = getComponent(instance, 'tags');
+  const extra = getComponent(instance, 'extra');
   const backIcon =
-    getComponentFromProp(instance, 'backIcon') !== undefined ? (
-      getComponentFromProp(instance, 'backIcon')
+    getComponent(instance, 'backIcon') !== undefined ? (
+      getComponent(instance, 'backIcon')
     ) : (
       <ArrowLeftOutlined />
     );
```
Comment (tangjinzhou) [inline components/page-header/index.jsx:64]: const onBack = instance.$listeners.back;
to
const onBack = instance.$attrs.onBack;

File: components/page-header/index.jsx (hunk 2 above)
Comment (xrkffgg) [reply components/page-header/index.jsx:107]: Sorry forget

File: components/page-header/index.jsx (hunk 1 above)
Comment (xrkffgg) [reply components/page-header/index.jsx:6]: ./components/breadcrumb/Breadcrumb.jsx
`Cannot read property 'includes' of undefined`

一直报这个错 @tangjinzhou

Comment (tangjinzhou) [reply components/page-header/index.jsx:6]: 因为有 <Xxx.Xxx /> 的使用方式  @Amour1688 插件不支持

Comment (Amour1688) [reply components/page-header/index.jsx:6]: 插件 bug，本周发版修复

**Pull request merged**